packages/flutter_tools/lib/src/commands/analyze_base.dart

```diff
@@ -85,6 +85,27 @@ abstract class AnalyzeBase {
   }
 
   bool get isBenchmarking => argResults['benchmark'] as bool;
+
+  String generateDartDocMessage(int undocumentedMembers) {
+    String dartDocMessage;
+
+    if (undocumentedMembers >= 0) {
```

Comment:
rather than using both a `switch` and an `if/else`, I would only `assert(undocumentedMembers >= 0)`.